core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -81,3 +81,15 @@
     {% endif %}
   </ul>
 </script>
+<style type="text/css">
+  ul .popover.bottom{
```

Comment:
Still to general, you will need apply a unique CSS class to that popover and label it here.